src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[][] data = readData(fromFileName);
+
+        String resultData = calculateStatistics(data);
+
+        writeData(toFileName, resultData);
+    }
+
+    public String[][] readData(String fileName) {
+        //Read from file
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+
+            StringBuilder stringBuilder = new StringBuilder();
+            String line;
+
+            while ((line = bufferedReader.readLine()) != null) {
+                stringBuilder.append(line).append(LINE_SEPARATOR);
+            }
+
+            String[] lines = stringBuilder.toString().split(LINE_SEPARATOR);
+            String[][] data = new String[lines.length][2];
+            for (int i = 0; i < lines.length; i++) {
+                data[i] = lines[i].split(",");
+            }
+            return data;
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from the file " + fileName, e);
+        }
+    }
+
+    public String calculateStatistics(String[][] data) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+        int result = 0;
```

Comment:
The variable `result` is redundant because it's just the difference between `totalSupply` and `totalBuy`, which can be calculated directly when building the result string.